tests/integration_tests/scheduler/bin/bhist.py

```diff
@@ -29,7 +30,7 @@ def get_parser() -> argparse.ArgumentParser:
         description="Mocked LSF bhist command reading state from filesystem"
     )
     parser.add_argument("-n", type=int, default=None)
-    parser.add_argument("-l", action="store_true")
+    parser.add_argument("-l", "--long-format", action="store_true")
```

Comment:
In the real `bhist`, `-l` and `--long-format` actually does different things..

Reply:
Or, more probably "`--long-format`" does not do what you expect it to do, the option parser of `bhist` is not acting similarly to python so it just does not work.

Reply:
I used it more like an alias for l, because `if args.long_format: ...` is more explanatory than `if args.l: ...`. I will remove it so that it wont create confusion in the future.